core/src/main/java/feast/core/service/AccessManagementService.java

```diff
@@ -42,11 +42,16 @@
   private AuthorizationProvider authorizationProvider;
   private ProjectRepository projectRepository;
 
+  // TODO: Remove duplication of constructor
   public AccessManagementService(
       FeastProperties feastProperties,
       ProjectRepository projectRepository,
       AuthorizationProvider authorizationProvider) {
     this.projectRepository = projectRepository;
+    // create default project if it does not yet exist.
+    if (!projectRepository.existsById(Project.DEFAULT_NAME)) {
```

Comment:
I wonder if this should be a part of our migration scripts instead of code.

Reply:
Added migration to create the default project.